flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobServerRangeTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.blob;
+
+import org.apache.flink.configuration.ConfigConstants;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.util.NetUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * Tests to ensure that the BlobServer properly starts on a specified range of available ports.
+ */
+public class BlobServerRangeTest {
+
+
+	/**
+	 * Start blob server on 0 = pick an ephemeral port
+	 */
+	@Test
+	public void testOnEphemeralPort() {
+		Configuration conf = new Configuration();
+		conf.setString(ConfigConstants.BLOB_SERVER_PORT, "0");
+		try {
+			BlobServer srv = new BlobServer(conf);
+		} catch (IOException e) {
```

Comment:
Why don't you let the exception propagate. This will be enough to fail the test and you will see the stack trace at the proper position.